mobile/routers/components/CustomDrawnerContent.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import { DrawerContentScrollView, DrawerItem, DrawerItemList } from '@react-navigation/drawer';
+import { View } from 'react-native';
+import AttractionSubMenu from '../../components/views/Attractions/Router';
+import TestSubMenu from '../../components/views/test/Router';
+
+
+const CustomDrawerContent: React.FC<any> = (props) => {
```

Comment:
I would call it `AttractionsDrawer` or something like this. `CustoDrawer` tells nothing about it